app/Console/Commands/SortLanguageFiles.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class SortLanguageFiles extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'language:sort';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Sort language expressions';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $languages = array_diff(scandir(base_path('resources/lang/')), ['..', '.']);
+        foreach ($languages as $language) {
+            $files = array_diff(scandir(base_path('resources/lang/'.$language)), ['..', '.']);
+            foreach ($files as $file_in) {
+                $expressions = require base_path('resources/lang/'.$language.'/'.$file_in);
+                if (! (ksort($expressions))) {
+                    $this->error('Sorting '.$file_in.' failed.');
+                }
+                $file_out = fopen(base_path('resources/lang/'.$language.'/'.$file_in), 'w');
```

Comment:
Indentation should be set, otherwise everyone could get different results. When I run the command, I got the results with simple braces () and 2 indentation, opposing to what we have in the diff now.

Reply:
Try it now

Reply:
Now, this function and StyleCI working against each other with the commas (see https://github.com/luksan47/mars/pull/245/commits/72f8ed63321a0e127fccdafe2bb707ae107b0287 and https://github.com/luksan47/mars/pull/245/commits/ebcf5a21699f7625af065c2c14da84310d92cacd)

Reply:
> Now, this function and StyleCI working against each other with the commas (see [72f8ed6](https://github.com/luksan47/mars/commit/72f8ed63321a0e127fccdafe2bb707ae107b0287) and [ebcf5a2](https://github.com/luksan47/mars/commit/ebcf5a21699f7625af065c2c14da84310d92cacd))

I fixed it on this side. Now a comma will be generated after the last element of an array.

Reply:
Great, thanks!

Reply:
The command runs fine for me now